autotest/ogr/ogr_pgeo.py

```diff
@@ -286,7 +286,101 @@ def test_ogr_pgeo_8():
     feat_count = non_spatial_layer.GetFeatureCount()
     assert feat_count == 2, 'did not get expected feature count'
 
+##################################################################################
+# Open mdb with polygon layer containing a mix of single and multi-part geometries
+
+
+def test_ogr_pgeo_9():
+    if ogrtest.pgeo_ds is None:
+        pytest.skip()
+
+    if ogrtest.active_driver.GetName() != 'PGeo':
+        # MDB driver doesn't report non-spatial tables
+        pytest.skip()
+
+    ogrtest.pgeo_ds = ogr.Open('data/pgeo/mixed_types.mdb')
```

Comment:
the practice of assigning dataset handles to a ogrtest. member should not be perpetuated. It makes potentially the tests interdependent which should be avoided Just use a ``ds`` local variable